test/smoke/aomp-issue376/aomp-issue376.cpp

```diff
@@ -42,8 +45,3 @@ int main()
 
 /// CHECK: Callback Target
 /// CHECK-SAME: device_num=0
```

Comment:
Assumption: Since we build with OMPT by default there's no real interest to support OMPT-disabled builds w.r.t. the smoke suites.

Also, was this the only fail of that kind?
When grepping for `for (auto Dev : *DeviceMapPtr)` there should be 9 `smoke*` tests (C-source files) with that particular issue.

Reply:
I used the smoke suite for validating that the downstream refactored version also works with OMPT disabled.
And, I may be wrong, but I _think_ this test was the only one that bailed out with a seg fault.

Reply:
Check, thanks JP!